enterprise/web.core/src/org/netbeans/modules/web/core/api/JspColoringData.java

```diff
@@ -118,9 +118,9 @@ public void parsingStarted() {
     /** Incorporates new parse data from the parser, possibly firing a change about coloring.
      * @param newTaglibs the new map of (uri -> TagLibraryInfo)
      * @param newPrefixMapper the new map of (prefix, uri)
-     * @param parseSuccessful wherher parsing was successful. If false, then the new information is partial only
+     * @param parseSuccessful wheher parsing was successful. If false, then the new information is partial only
```

Comment:
Should it be _whether_?

Reply:
Hey @tonihele , yep.. Good eyes.. 8-)

Fixed.